goosepaper/multiparser.py

```diff
@@ -27,35 +37,150 @@ def __init__(self):
             required=False,
             help="The output file path at which to save the paper",
         )
+        self.parser.add_argument(
+            "-f",
+            "--folder",
+            required=False,
+            help="Folder to which the document will be uploaded in your remarkable.",
+        )
         self.parser.add_argument(
             "-u",
             "--upload",
             action="store_true",
             required=False,
+            default=None,
             help="Whether to upload the file to your remarkable using rmapy.",
         )
+        self.parser.add_argument(
+            "--noupload",
+            action="store_true",
+            required=False,
+            default=None,
+            help="Overrides any other 'upload: true' arguments from config files or command line. Useful for testing configs or story generation without having to edit config files."
+        )
+        self.parser.add_argument(
+            "--showconfig",
+            action="store_true",
+            required=False,
+            default=None,
+            help="Print out all config files and command line options in order loaded and the final config to help finding conflicting options. Needed since there are now four possible ways to pass options."
+        )
+        self.parser.add_argument(
+            "-n",
+            "--nostory",
+            required=False,
+            default=False,
+            action="store_true",
+            help="||Skip story creation. Combined with \"--upload\" can be used to\nupload a preexisting output file.\n\n** NOTE ** If used without \"--upload\" goosepaper will run but\nperform no action.",
```

Comment:
Not totally convinced this option makes sense here — more on that in the review notes!

Reply:
I still like it. It makes testing config parse order easier when combined with '--showconfig'. I HAD to put it in in order to make validating why options were conflicting given four different mechanisms for consuming options.
:)